docs/apps/components/radio.md

```diff
@@ -0,0 +1,178 @@
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of the radio component in response to user input.
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+    4. Before using **this.enable(fields)**, make sure the component is **disabled**. This can be done using **this.disable(fields)** or by **enabling** the **Disabled checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/radio1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('radio_mv3o2rpn', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('radio_mv3o2rpn', 'abc')
+        ```
+    
+    3. `this.getValues()`
+        ```js
+        var data = this.getValues();
+        console.log(data);
+        ```
+    
+    4. `this.hide(fields)`
+        ```js
+        var fields= ['radio_mv3o2rpn']
+        this.hide(fields)
+        ```
+    
+    5. `this.show(fields)`
+        ```js
+        var fields= ['radio_mv3o2rpn']
+        this.show(fields)
+        ```
+    
+    6. `this.disable(['fields'])`
+        ```js
+        this.disable(['radio_mv3o2rpn'])
+        ```
+    
+    7. `this.enable(['fields'])`
+        ```js
+        this.enable(['radio_mv3o2rpn'])
+        ```
+    
+    8.  `this.getValue('fieldName')`
+        ```js
+        var radioname = this.getValue('radio_mv3o2rpn');
+        console.log('getValue', radioname);
+        ```
+
+    9.  `this.getComponent('component_ID')`
+        ```js
+        var radioname = this.getComponent('radio_mv3o2rpn');
+        console.log('getComponent', radioname);
+        ```
+
+    10.  `this.setData(Value)`
+        ```js
+        this.setData(
+         {"radio_mv3o2rpn": "Option 3",});
+        ```
+
+**example of refreshfield and getfield**
```

Comment:
I am not sure if this should be here. Because there are no refreshfield and getfield examples. Also the functions have wrong names.

Reply:
I have removed it from the documentation for the time being.. will confirm from Saurabh and then add if its applicable.